core/src/main/resources/commonsdivision/article/index.scala.html

```diff
@@ -104,7 +106,7 @@ <h4 class="atom--snippet__headline">'@Html(title)'</h4>
     </summary>
 
     <div class="atom--snippet__body atom--commonsdivision__body">
-      @{partiesTable(data.votes.ayes.groupBy(_.party), data.votes.noes.groupBy(_.party))}
+      @{partiesTable(data.votes.ayes.toSeq.groupBy(_.party), data.votes.noes.toSeq.groupBy(_.party))}
```

Comment:
Can you tell me how is this related to removing storyquestions?

Reply:
I upgraded to scala 2.13 at the same time ✌️